examples/provider/provider.tf

```diff
@@ -0,0 +1,21 @@
+terraform {
+  required_providers {
+    anxcloud = {
+      source  = "anexia-it/anxcloud"
+      version = "~> 0.3"
+    }
+  }
+}
+
+// Authentication via environment variable is strongly advised:
+// export ANEXIA_TOKEN='<token>'
+
+// Alternatively, but NOT recommended:
+provider "anxcloud" {
+  token = "<token>"
+}
+
+
+data "anxcloud_core_location" "anx04" {
+  code = "ANX04"
+}
```

Comment:
maybe we should extend the provider example to a full workflow of searching the location, creating a vlan, a public v4 and v6 prefix, addresses, searching a template and creating a VM?

Would allow making the examples on the specific resources and data sources simpler, when using variables here for that stuff, the examples on specific resources/data sources are clearer

Reply:
Unfortunately template handling is still only possible with the `anxcloud_template` which is actually a plural (list) data-source. I think we should have a new data-source which only holds a single template. SYSENG-792 would also be good to have.

Reply:
I've extended the provider example to use more resources and data-sources. I'll add an example for IPv6 with SYSENG-1236.